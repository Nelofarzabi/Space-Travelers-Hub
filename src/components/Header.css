header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  background-color: whitesmoke;
  padding: 20px;
}

.header--logo {
  display: flex;
  align-items: center;
  gap: 1rem;
}

.link--list {
  list-style: none;
  display: flex;
  gap: 0.7rem;
}

.link--list a {
  color: rgba(0, 0, 255, 0.637);
  font-size: 1.1rem;
}