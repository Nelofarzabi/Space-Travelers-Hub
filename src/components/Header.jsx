import React from 'react'
import planet from '../assets/planet.png'
const Header = () => {
  return (
    <header>
      <nav>
        <ul>
          <li></li>
        </ul>
      </nav>
    </header>
  )
}

export default Header