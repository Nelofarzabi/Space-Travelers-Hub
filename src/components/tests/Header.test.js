import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Header from '../Header';

test('renders the header with navigation links', () => {
  const { container } = render(
    <MemoryRouter>
      <Header />
    </MemoryRouter>,
  );
  expect(container).toMatchSnapshot();
});
