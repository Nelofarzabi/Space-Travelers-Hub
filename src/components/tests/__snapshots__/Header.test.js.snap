// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the header with navigation links 1`] = `
<div>
  <header>
    <section
      class="header--logo"
    >
      <img
        alt="planet logo"
        src="planet.png"
      />
      <h1>
        Space Travelers' Hub
      </h1>
    </section>
    <nav>
      <ul
        class="link--list"
      >
        <li>
          <a
            aria-current="page"
            class="active"
            href="/"
            style="text-decoration: underline;"
          >
            Rockets
          </a>
        </li>
        <li>
          <a
            class=""
            href="/missions"
          >
            Missions
          </a>
        </li>
        <li>
          <a
            class=""
            href="/profile"
          >
            My Profile
          </a>
        </li>
      </ul>
    </nav>
  </header>
</div>
`;
